docs/source/workflow.rst

```diff
@@ -0,0 +1,19 @@
+.. include:: /alert-noindex.rst
```

Comment:
This shouldn't be included here (inserts the pre-alpha disclaimer and excludes page from google robots)